packages/react-core/src/demos/DataList/examples/DataListBasic.tsx

```diff
@@ -140,23 +150,41 @@ export const DataListBasic: React.FunctionComponent = () => {
                         <FlexItem>
                           <Flex spaceItems={{ default: 'spaceItemsSm' }}>
                             <FlexItem>
-                              <CodeBranchIcon /> 5
+                              <Icon>
+                                <CodeBranchIcon />
+                              </Icon>
+                              5 <span className="pf-v6-screen-reader">Branches</span>
                             </FlexItem>
                             <FlexItem>
-                              <CodeIcon /> 9
+                              <Icon>
+                                <CodeIcon />
+                              </Icon>
+                              9 <span className="pf-v6-screen-reader">Code blocks</span>
                             </FlexItem>
                             <FlexItem>
-                              <CubeIcon /> 2
+                              <Icon>
+                                <CubeIcon />
+                              </Icon>
+                              2 <span className="pf-v6-screen-reader">Workspaces</span>
                             </FlexItem>
                             <FlexItem>
-                              <CheckCircleIcon />
+                              <Icon status="success">
+                                <CheckCircleIcon />
+                              </Icon>
                               11
+                              <span className="pf-v6-screen-reader">Completed</span>
                             </FlexItem>
                             <FlexItem>
-                              <ExclamationTriangleIcon /> 4
+                              <Icon status="warning">
+                                <ExclamationTriangleIcon />
+                              </Icon>
+                              4<span className="pf-v6-screen-reader">Warnings</span>
                             </FlexItem>
                             <FlexItem>
-                              <TimesCircleIcon /> 1
+                              <Icon status="danger">
+                                <ExclamationCircleIcon />
+                              </Icon>
+                              1<span className="pf-v6-screen-reader">Errors</span>
```

Comment:
Does this need to be singular?

```suggestion
                              1<span className="pf-v6-screen-reader">Error</span>
```

Reply:
Ah copy+paste strikes back again